x-pack/legacy/plugins/ml/public/data_frame_analytics/pages/analytics_management/components/analytics_list/expanded_row.tsx

```diff
@@ -235,19 +235,16 @@ export const ExpandedRow: FC<Props> = ({ item }) => {
       name: 'JSON',
       content: <ExpandedRowJsonPane json={item.config} />,
     },
-    // Audit messages are not yet supported by the analytics API.
-    /*
     {
       id: 'ml-analytics-job-messages',
       name: i18n.translate(
         'xpack.ml.dataframe.analyticsList.analyticsDetails.tabs.analyticsMessagesLabel',
         {
-          defaultMessage: 'Messages',
+          defaultMessage: 'Job messages',
         }
       ),
       content: <ExpandedRowMessagesPane analyticsId={item.id} />,
```

Comment:
Looks like you need to copy across some of the styles from the anomaly detection job messages table. The columns aren't aligning or sizing as they should:

Analytics job:
![image](https://user-images.githubusercontent.com/7405507/68776658-60c1e600-0628-11ea-9630-c3dbef1b5b35.png)

Anomaly detection job:
![image](https://user-images.githubusercontent.com/7405507/68776702-746d4c80-0628-11ea-998f-5842823c3ce0.png)

Reply:
Now both tables rendred by the same component, check https://github.com/elastic/kibana/pull/50468/commits/a63573d32fded9f51c711d2423b4e503b27532af